.babelrc

```diff
@@ -1,4 +1,7 @@
 {
-  "plugins": ["transform-class-constructor-call"],
-  "presets": ["es2015", "stage-2"]
+  "presets": ["es2015", "stage-2"],
```

Comment:
We still need `stage-2` but this is the only project that seems to do this. Will follow up on why this is necessary in: https://globality.atlassian.net/browse/GLOB-18988 and remove if possible.

Reply:
I'm using "env" as the only preset now.